lib/puppet/type/mount.rb

```diff
@@ -275,6 +275,23 @@ def munge(value)
       end
     end
 
+    newparam(:manage_mountpoint) do
```

Comment:
You can add `:boolean => true` here, which will get us the use of `manage_mountpoint?`, courtesy of https://github.com/puppetlabs/puppet/blob/327b03a9e1ecdc7b925720a4eec59795f4456a56/lib/puppet/type.rb#L283-L293